external/protobuf/.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "third_party/benchmark"]
```

Comment:
Is this submodule used?

Reply:
@sunghan-chang This is not using now, but it will be used to check performance of protobuf later.